docs-source/examples.rst

```diff
@@ -0,0 +1,250 @@
+.. globus-cli examples master file
+
+Globus CLI Examples
+===================
+
+.. note::
+    The following examples are all written for a Bash shell, if you are running
+    another shell you may need to modify the non globus commands and syntax
+    found in these examples
+
+    The following examples also assume the user has already successfully 
+    authenticated with Globus Auth. To do so run
+
+    .. code-block:: bash
+
+        $ globus login
+
+    and follow the instructions. 
+    If you are having problems with authorization, consult `Getting Started`_
+
+    .. _Getting Started: https://globus.github.io/globus-cli/# getting-started
+
+
+Finding an Endpoint
+-------------------
+
+Searches for a tutorial endpoint, then stores its ID in a local variable to
+allow for human-readable calls to further globus endpoint commands.
+
+.. code-block:: bash
+    
+    $ globus endpoint search "Globus Tutorial Endpoint"
```

Comment:
This applies throughout, but could you include sample (text-mode) output? I think it would make the examples more readable, especially in cases like this where we derive a value used below.

It's okay to doctor the output to look nicer (e.x. replace `ls` results with an invented set of files/dirs).